distribution/openrct2.d.ts

```diff
@@ -1582,16 +1583,23 @@ declare global {
         direction: Direction;
         object: number;
         primaryColour: number;
-        secondaryColour: number;
+        secondaryColour: number; // text colour if element is banner
```

Comment:
I'd recommend using JSDoc as it has better support than postfixed comments in programs like VSCode.

```suggestion
        /**
         * If the object is a banner, this will be the text colour.
         */
        secondaryColour: number; 
```